functions.php

```diff
@@ -126,8 +126,31 @@ function thunderblog_customize_register ( $wp_customize ) {
 		)
 	);
 }
+
+/**
+ * Adds the featured image (if any) to the (rss/atom/whatever) feed.
+ *
+ * @param string $text
+ *
+ * @return string
+ */
+function thunderblog_add_featured_image_to_feed( string $text ): string {
+	global $post;
+
+	// No featured image? Skip!
+	if ( ! has_post_thumbnail( $post ) ) {
+		return $text;
+	}
+
+	$featured_image = get_the_post_thumbnail( $post, [ 640, 360 ] );
+
+	return "<p>{$featured_image}</p>{$text}";
```

Comment:
Since this is a `filter`, we have to return the first parameter. This will also modify it for any additional filters / code.